src/js/themes/base.js

```diff
@@ -274,7 +276,12 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         },
       },
       input: {
-        padding: `${baseSpacing / 2}px`,
```

Comment:
Just checking, did we have to change this? I'm assuming the snapshots would have aligned either way.

Reply:
No, we didn't need to change it. Just was updating based on conversation that it was okay to change base.js to use an object.

Reply:
why are we changing it if we don't have to?

Reply:
I am fine leaving it as it was, and just allowing grommet to support an object if a theme (such as grommet-theme-hpe) wanted to do that. I'll do that.

Reply:
Actually, for backwards compatibility we have a check in styles L281 to maintain backwards compatibility for themes using just a string. However, we do need to update base.js to be an object.